attendance/API_HR/api.php

```diff
@@ -656,6 +656,12 @@
     $res = HR::api_addInventoryAudit( $inventory_id, $logged_user_id, $audit_message );
 }
 
+else if( $action == 'get_inventory_audit_status_month_wise' ){
+    $month = $PARAMS['month'];
```

Comment:
if no month is passed, it should have current month,  
there is an existing function to get current, previous and next month